components/pedigree/resources/src/main/resources/pedigree/pedigree.js

```diff
@@ -621,6 +621,16 @@ define([
         },
 
         /**
+         * Returns a list of all available legends
+         */
+        getAllLegends: function() {
+            return [ this.getCancerLegend(),
+                     this.getHPOLegend(),
+                     this.getDisorderLegend(),
+                     this.getCarrierGeneLegend(), this.getRejectedGeneLegend(), this.getCandidateGeneLegend(), this.getCausalGeneLegend() ];
+        },
```

Comment:
agree, but this is a larger fix than this issue name implies, I will create a new issue for that and think a bit how to do it right instead of doing something better but still a quick-fix-type (may still make it to m1, but will assign m2 for now). The change here at the very least makes sure all the _other_ code only uses this one method instead of manually calling 5 legends (as it used to, though only in one place - but now there are more places)